.storybook/blocks/ComponentListItem/ComponentListItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactNode } from 'react';
+
+import { Status } from '../Status';
+
+import styles from './ComponentListItem.module.css';
+
+type ComponentListItemProps = {
+  name: string;
+  version?: string;
+  stage: number;
+  status?: string;
+  design?: string;
+};
+
+export const ComponentListItem: React.FC<ComponentListItemProps> = ({
+  name,
+  version,
+  stage,
+  design,
+}) => {
+  const packageName = name
+    .replace(/([a-z0-9])([A-Z])/g, '$1-$2')
+    .replace(/([A-Z])([A-Z])(?=[a-z])/g, '$1-$2')
+    .toLowerCase();
+  const githubLink = `https://github.com/hey-car/heycar-uikit/tree/main/packages/${packageName}`;
+
+  return (
+    <div className={styles.component}>
+      <div className={styles.details}>
+        <h3>
+          <a
+            href={
+              name === 'Fonts'
+                ? '/?path=/docs/guidelines-fonts--page'
+                : `/?path=/docs/components-${name}--${name}`
+            }
+          >
+            {name}
+          </a>
+        </h3>
+        <span className={styles.version}>{version}</span>
```

Comment:
Can we add npm package link here? https://www.npmjs.com/package/@heycar-uikit/button